src/bindings/python/tests_compatibility/test_utils/test_utils.py

```diff
@@ -2,42 +2,36 @@
 # SPDX-License-Identifier: Apache-2.0
 
 import openvino
-from openvino.runtime import Model, Core, Shape, Type
-from openvino.runtime.op import Parameter
+import ngraph as ng
+from openvino.inference_engine import IECore, IENetwork
+from ngraph.impl import Function, Shape, Type
+from ngraph.impl.op import Parameter
+
 from typing import Tuple, Union, List
-import openvino.runtime.opset8 as ops
 import numpy as np
 
 
 def get_test_function():
     element_type = Type.f32
     param = Parameter(element_type, Shape([1, 3, 22, 22]))
-    relu = ops.relu(param)
-    func = Model([relu], [param], "test")
+    relu = ng.relu(param)
+    func = Function([relu], [param], "test")
     assert func is not None
     return func
 
 
-def test_compare_functions():
-    try:
-        from openvino.test_utils import compare_functions
-        func = get_test_function()
-        status, _ = compare_functions(func, func)
-        assert status
-    except ModuleNotFoundError:
-        print("openvino.test_utils.compare_functions is not available")
```

Comment:
Should this test be deleted? I remember it causing issues due to import errors.

Reply:
this function is used in e2e tests but from the `src/bindings/python/tests/test_utils/test_utile.py` file, so here itsn't need at all.